### DISCUSSION TITLE
how to disable you can't injure a defenseless player if they do not have any item in their inventory

### DISCUSSION THREAD

**Discussion by jadeisalone** [GriefPrevention/GriefPrevention#2002] [Q&A]
how to disable you can't injure a defenseless player if they do not have any item in their inventory in the grief prevention plugin Minecraft

**Comment by bigpresh** [comment]
https://docs.griefprevention.com/configuration/#pvp-configuration - 

> `PvP.ProtectFreshSpawns: true`
>
> Set this to false to disable the automatic no-PvP status for new players and fresh respawns. The immunity lasts at least 10 seconds, during which they can’t pick up items (so they can’t abuse the invincibility to collect loot and run away), then disappears when they pick up an item the initial 10 second period. This effectively stops spawn camping in all cases, not only when everyone shares a designated no-pvp respawn area.
